usecase/pokemon/pokemon.go

```diff
@@ -0,0 +1,72 @@
+package pokemon
```

Comment:
Just like its common for controllers to go on a single package, it also applies for usecases. The only things that should go on different folders are services.

You can have just one usecase folder and separate common logic in their own files.

Reply:
*